scenes/ink_combat/ink_combat.gd

```diff
@@ -12,6 +13,12 @@ var inkwells_completed: int = 0
 
 
 func _ready() -> void:
+	DialogueManager.show_dialogue_balloon(intro_dialogue, "", [self])
+	await DialogueManager.dialogue_ended
+	# Add a short delay so the player doesn"t attack when closing the dialogue:
+	await get_tree().create_timer(0.5).timeout
```

Comment:
Not necessarily one for this branch but it would be really nice to fix this more systematically rather than needing a timeout after everywhere that we have dialogue. I think moving player input handling to `_unhandled_input()` would do it.